tools-image/build-arm-image.sh

```diff
@@ -223,6 +223,7 @@ else
 fi
 
 if [ -n "$cos_config" ] && [ -e "$cos_config" ]; then
+  # shellcheck source=/dev/null
```

Comment:
The hell? What does this do?

Reply:
Shellcheck tries to validate the file you're sourcing. But because in this case it can be a file given at runtime we just tell it to check no file by pointing there